NPE in analyser plugin

When running the Analyser plugin on a 'plain vanilla' project the following exception appears:

```
Caused by: java.lang.NullPointerException
    at java.util.ArrayList.addAll (ArrayList.java:702)
    at com.adobe.aem.analyser.AemAnalyser.getTierMessages (AemAnalyser.java:281)
    at com.adobe.aem.analyser.AemAnalyser.logOutput (AemAnalyser.java:291)
    at com.adobe.aem.analyser.AemAnalyser.analyse (AemAnalyser.java:220)
    at com.adobe.aem.analyser.mojos.AemAnalyseMojo.analyseFeatures (AemAnalyseMojo.java:303)
    at com.adobe.aem.analyser.mojos.AemAnalyseMojo.execute (AemAnalyseMojo.java:209)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137
```

--- a/aemanalyser-maven-plugin/src/main/java/com/adobe/aem/analyser/AemAnalyser.java
+++ b/aemanalyser-maven-plugin/src/main/java/com/adobe/aem/analyser/AemAnalyser.java
@@ -15,6 +15,7 @@ import java.io.IOException;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
 import java.util.HashMap;
 import java.util.LinkedHashMap;
 import java.util.LinkedHashSet;
@@ -278,9 +279,9 @@ public class AemAnalyser {
         List<String> msgs = messages.get(tier);
         if ( msgs == null ) {
             msgs = new ArrayList<>();
-            msgs.addAll(messages.get(tier.concat(".dev")));
-            msgs.retainAll(messages.get(tier.concat(".stage")));
-            msgs.retainAll(messages.get(tier.concat(".prod")));
+            msgs.addAll(messages.getOrDefault(tier.concat(".dev"), Collections.emptyList()));
+            msgs.retainAll(messages.getOrDefault(tier.concat(".stage"), Collections.emptyList()));
+            msgs.retainAll(messages.getOrDefault(tier.concat(".prod"), Collections.emptyList()));
             messages.put(tier, msgs);
         }
         return msgs;

